### PR TITLE
add hasta test instructions to PR template

### PR DESCRIPTION
**Pull request by dnil**
This PR adds documentation.

**How to test**:
1. use test instructions on hasta

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -15,7 +15,19 @@ This PR marks a new Scout release. We apply semantic versioning. This is a major
...
 1. Make sure the branch is deployed: `systemctl --user status scout.target`
-1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
+1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
+</details>
+
+<details>
+<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>
+
+**Prepare for testing**
+1. `ssh <USER.NAME>@hasta.scilifelab.se`
```
Comment (northwestwitch) [inline .github/PULL_REQUEST_TEMPLATE.md:25]: Switch poin 1 and point 2?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -15,7 +15,19 @@ This PR marks a new Scout release. We apply semantic versioning. This is a major
 1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
 1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
 1. Make sure the branch is deployed: `systemctl --user status scout.target`
-1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
+1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
```
Comment (northwestwitch) [inline .github/PULL_REQUEST_TEMPLATE.md:18]: Replace the paxa command runned oin hasta with the button at https://pax.scilifelab.se, also above in this list, when the paxa command is called before testing

**Review by northwestwitch — APPROVED** [review]
👍🏻 I have a couple of suggestions!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -15,7 +15,19 @@ This PR marks a new Scout release. We apply semantic versioning. This is a major
...
+</details>
+
+<details>
+<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>
+
+**Prepare for testing**
+1. `ssh <USER.NAME>@hasta.scilifelab.se`
+1. Book your testing time using the Pax software. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `hasta`. `us; paxa -u <user> -s hasta -r scout-stage`.
+1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
+1. Stop the service with current deployed branch: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b<this_branch>`
```
Comment (northwestwitch) [inline .github/PULL_REQUEST_TEMPLATE.md:28]:
```suggestion
1. Stop the service with current deployed branch: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (dnil) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: This is intentional: much easier to use the cli when one is anyway on hasta? No need to go to a second system. But you are very right about the above section for cg-vm1!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (dnil) [reply .github/PULL_REQUEST_TEMPLATE.md:28]: Hmm, I thought I did that already? 😊

Comment (henrikstranneheim) [reply .github/PULL_REQUEST_TEMPLATE.md:28]: Is it correct to say `stop the service` here or just deploy your PR branch or similar?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -15,7 +15,19 @@ This PR marks a new Scout release. We apply semantic versioning. This is a major
...
+<details>
+<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>
+
+**Prepare for testing**
+1. `ssh <USER.NAME>@hasta.scilifelab.se`
+1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
+1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
+1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
+1. Make sure the branch is deployed: `us; scout --version`
+1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
```
Comment (northwestwitch) [inline .github/PULL_REQUEST_TEMPLATE.md:30]: Sorry, it just came to my mind. perhaps another step here that illustrates how to test? example:

scout --config /home/proj/stage/servers/config/hasta.scilifelab.se/scout-stage.yaml <command>

Comment (northwestwitch) [reply .github/PULL_REQUEST_TEMPLATE.md:30]: whoops, too late. No worries, another time!

Comment (dnil) [reply .github/PULL_REQUEST_TEMPLATE.md:30]: Or say it's a servers PR in that case. 😸

**Pull request merged**